api/v2alpha1/subnamespace_types.go

```diff
@@ -0,0 +1,59 @@
+package v2alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// SubNamespaceStatus defines the observed state of SubNamespace
+// +kubebuilder:validation:Enum=ok;conflict
+type SubNamespaceStatus string
+
+const (
+	SubNamespaceOK       = SubNamespaceStatus("ok")
+	SubNamespaceConflict = SubNamespaceStatus("conflict")
+)
+
+// SubNamespaceSpec defines the desired state of SubNamespace
+type SubNamespaceSpec struct {
+	// Labels are the labels to be propagated to the sub-namespace
+	// +optional
+	Labels map[string]string `json:"labels,omitempty"`
+
+	// Annotations are the annotations to be propagated to the sub-namespace.
+	// +optional
+	Annotations map[string]string `json:"annotations,omitempty"`
+}
+
+// Keeping this version un-served for now
+//+kubebuilder:unservedversion
+//+kubebuilder:object:root=true
```

Comment:
I guess we're going to make the `status` a subresource.
```suggestion
// Keeping this version un-served for now
// +kubebuilder:unservedversion
// +kubebuilder:object:root=true
// +kubebuilder:subresource:status
```

Reply:
I was planning to start with an _identical_ new version, but we are heading for a status subresource. Do you prefer heading for https://github.com/cybozu-go/accurate/pull/96 directly?

Reply:
I understand.